crowbar_framework/app/models/node_object.rb

```diff
@@ -1169,7 +1176,7 @@ def set_state(state)
         Rails.logger.warn("chef client seems to be still running after 5 minutes of wait; going on with the reboot")
       end
 
-      bmc_cmd("power cycle")
+      ssh_cmd("/sbin/reboot")
```

Comment:
What about this? If we call it only ```shutdown -r``` it should work on windows and linux platform.